packages/react-native-codegen/scripts/oss/build.sh

```diff
@@ -11,9 +11,18 @@ THIS_DIR=$(cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOUR
 set -e
 set -u
 
-pushd "$THIS_DIR/../.." >/dev/null
+TMP_DIR=$(mktemp -d)
+CODEGEN_DIR="$THIS_DIR/../.."
+
+rm -rf "$CODEGEN_DIR/lib" "$CODEGEN_DIR/node_modules"
+
+cp -R "$CODEGEN_DIR/." "$TMP_DIR"
```

Comment:
just to make sure this is compatible with MacOS or Linux (circle CI etc), let's use `rsync` if we can

https://github.com/facebook/react-native/blob/master/packages/react-native-codegen/DEFS.bzl#L80